### ISSUE TITLE
Find a better solution when code_linkage() can not identify the "mother" treaty

### ISSUE THREAD

**Issue by EstherPeev** [globalgov/qCreate#25]
In many cases, `code_linkages()` can't find the "mother" treaty in the dataset, so it is linking the amendment/protocol to another protocol/amendment. It is either because the "mother" treaty is not in the dataset or that the title is too different for the function to recognize them as part of the same family.

 [https://app.zenhub.com/files/365543800/54ed124a-8a1d-42cf-b9f6-23a19930f33f/download](https://app.zenhub.com/files/365543800/54ed124a-8a1d-42cf-b9f6-23a19930f33f/download)

**Comment by henriquesposito** [comment]
Good catch! 

I think we can deal with this by by removing returns from for the line value that are not agreements (keep only type A, bilateral treaties or known agreements) in code_linkages().  

I will take a look at this issue, thank you!

**Comment by henriquesposito** [comment]
I have removed returns that are amendments or protocols from code_linkages(), this should take care of the issue of linking the amendment/protocol to another protocol/amendment.

As for improving mother treaty identification, I have updated the predictable words list to include some of the misses illustrated. We will need to update this list and add new tests for code_agreements() when we notice these issues again in the future.
